### PR TITLE
task 4 added

### PR DESCRIPTION
**Pull request by SvetSkills**
task 4
comments: in spec file I've miss blank line one more time(

### CODE REVIEW

File: tasks1-10/task4.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Four
+  def largestPalindromeOfThreeDigits()
```
Comment (AndreyAnischik) [inline tasks1-10/task4.rb]: in ruby we use snake case and braces are redundant if you pass no args

File: tasks1-10/task4.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    boundary = 10**3 - 1
```
Comment (AndreyAnischik) [inline tasks1-10/task4.rb]: why we need this calculation? why not to use 999

File: tasks1-10/task4.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        extension = initial
+        reverse = 0
+        while extension != 0 do
+          reverse = reverse * 10 + extension % 10
+          extension /= 10
+        end
+        if initial == reverse && initial > max
+          max = initial
+        end
+        end
```
Comment (AndreyAnischik) [inline tasks1-10/task4.rb]: indentation

File: tasks1-10/task4.rb (hunk 2 above)
Comment (SvetSkills) [reply tasks1-10/task4.rb]: to change 3 for another num if it's necessary
 ok, I'll write code only to the specific task

File: tasks1-10/task4.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+class Four
+  def largest_palindrome_3_digits
+    boundary = 999
```
Comment (AndreyAnischik) [inline tasks1-10/task4.rb]: make boundary and lowest variables to be constants and move them out of function

**Review by AndreyAnischik — APPROVED** [review]

**Pull request merged**